### PR TITLE
Added solution for the task 7

### PR DESCRIPTION
**Pull request by sayumikko**
- [x] Найти данные в виде изображений для задачи классификации. Данные взяты отсюда: http://ufldl.stanford.edu/housenumbers/.
 - [x] Реализовать классы, необходимые для построения сети со следующими слоями
         - FullyConnectedLayer
         - ReluLayer
         - FullyConnectedLayer
 - [x] Использовать CrossEntropyLoss и L2-регуляризацию.
 - [x] Обучить модель на тренировочных данных, подбирать параметры (особенно learning rate) на валидационной и оценить качество на тестовой. Анализировать графики train/val loss, проверять на каждом шаге корректность вычисления градиентов с помощью разностной оценки.
 - [x] * (+2 балла) Добавить Batch normalization.
 - [x] * (+2 балла) В качестве оптимизатор использовать один из: Momentum, RMSprop.
 - [x] * (+1 балл) Также реализовать оптимизатор Adam.

Решение находится в папке hw. Работу выполнила Кузьмина Ксения, 21.Б11-мм. Датасет загрузить не получилось из-за его размеров и ограничения гитхаба, однако использован рекомендованный в задании в формате mat.

### CODE REVIEW

**Review by rustam-azimov — APPROVED** [review]
Отлично, 10 баллов.
